### PR TITLE
Add a test for filter api to cover duplicate dimensions

### PR DESCRIPTION
**Pull request by mattrout92**
### What

Add a test for https://github.com/ONSdigital/dp-filter-api/pull/91

### How to review

Run tests, make sure they pass

### Who can review

Anyone

### CODE REVIEW

File: publishing/filterAPI/json.go [hunk 1]
```diff
@@ -627,9 +627,35 @@ func GetValidPUTFilterOutputWithDimensionsJSON() string {
 		  {
 			  "name": "age",
 			  "options": [
-			    "27", "28"
+			    "27", "27", "28"
```
Comment (nshumoogum) [inline publishing/filterAPI/json.go]: Is this a valid request body, won't the filter api respond with duplicate option?

Comment (nshumoogum) [reply publishing/filterAPI/json.go]: If this allows for duplicate options and collapses them so there is no duplication in our database then it might be worth passing in a list of options to this function and then it is more flexible to set up different tests (one with duplicate options and one without)?

**Review by nshumoogum — CHANGES_REQUESTED** [review]

**Review by nshumoogum — APPROVED** [review]
Neat 👍
